aea/crypto/ethereum.py

```diff
@@ -176,12 +182,18 @@ def __init__(self, address: str, gas_price: str = DEFAULT_GAS_PRICE):
         """
         self._api = Web3(HTTPProvider(endpoint_uri=address))
         self._gas_price = gas_price
+        self._contract_address = None
```

Comment:
The contract address has no reason to be here. The ledger api might interact with many contracts.

Reply:
I know. It is more of an experimental variable. Cause I was facing some problems with the contract and wanted to discuss it with you :)